src/components/cms/SkillCreator/SkillWizard.js

```diff
@@ -1030,6 +1060,38 @@ class SkillWizard extends Component {
             </a>
             <CenterDiv>
               <DropDownDiv>
+                <Popover
+                  open={open}
+                  anchorEl={anchorEl}
+                  onClose={this.handleClose}
+                  anchorOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'center',
+                  }}
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'center',
+                  }}
+                >
+                  <Text>
+                    Open Source Skills will show in the public directory and are
+                    editable by everyone just like a Wikipedia article. Private
+                    skills are only editable by the creator/owner of the skill.
+                    They are not listed in the public directory and can
+                    currently only be used by the creator/owner.
+                  </Text>
+                </Popover>
+                <DetailText>
+                  Open Source and Public Skill(Coming Soon) (
+                  <span
```

Comment:
Styled components?

Reply:
@samagragupta For less than equal two styles on a component no need of styled components, as the same style is not applied anywhere else on the page